src/app/controllers/authentication.server.controller.js

```diff
@@ -85,10 +85,31 @@ exports.validateDomain = function (req, res, next) {
   })
 }
 
+/**
+ * Returns the error message when a user cannot perform an action on a form
+ * @param {String} user -  user email
+ * @param {String} title -  form title
+ * @returns {String} - the error message
+ */
+const makeUnauthorizedMessage = (user, title) => {
+  return `User ${user} is not authorized to perform this operation on Form: ${title}.`
+}
+
+/**
+ * Logs an error message when a user cannot perform an action on a form
+ * @param {String} user -  user email
+ * @param {String} requiredPermission -  level of permission required
+ * @param {String} form -  form
+ * @returns {String} - the error message
+ */
+const logUnauthorizedAccess = (user, requiredPermission, form) => {
+  const msg = `User ${user.email} not authorized to perform ${requiredPermission} operation on Form ${form._id} with title: ${form.title}.`
+  logger.error(msg)
```

Comment:
similarly logger format needs to be changed here

Reply:
done